velox/type/Conversions.h

```diff
@@ -37,6 +37,19 @@ struct Converter {
   }
 };
 
+namespace {
+inline void checkNoWhitespace(const char* data, size_t size) {
+  if (size > 0) {
+    VELOX_USER_CHECK(
+        !std::isspace(data[0]),
+        "Cannot cast a string with leading space to an integral value.");
+    VELOX_USER_CHECK(
+        !std::isspace(data[size - 1]),
```

Comment:
There were some issues with exotic whitespaces recently.
I wonder if we need to detect non-ascii whitespaces here too?

Reply:
+1